webpack.config.js

```diff
@@ -220,5 +222,10 @@ function createEntry(entry) {
 // main = app
 // toolbar = toolbar
 // shared_dashboard = publicly available dashboard
-module.exports = () => [createEntry('main'), createEntry('toolbar'), createEntry('shared_dashboard')]
+module.exports = () => [
+    createEntry('main'),
+    createEntry('toolbar'),
+    createEntry('shared_dashboard'),
+    createEntry('exporter'),
+]
```

Comment:
Ever since we replaced webpack with esbuild for the main app, I think we only compile cypress and storybook via the `main` entrypoint in webpack. Thus, basically all the others could be *yeeted*, and there shouldn't be a need to add any new ones.

Reply:
I'll give it a go